pipeline/merger/src/main/scala/weco/pipeline/merger/Main.scala

```diff
@@ -43,9 +45,12 @@ object Main extends WellcomeTypesafeApp {
         namespace = "identified-works"
       )
     )
-
+    val mergerMode =
+      config.getStringOption("merger.rules.mode").getOrElse("default")
+    val mergerRules =
+      if (mergerMode == "tei") TeiPlatformMerger else DefaultPlatformMerger
```

Comment:
```suggestion
    val mergerRules = mergerMode match {
      case "tei" => TeiPlatformMerger
      case _ => DefaultPlatformMerger
    }
```

Using pattern-matching would make it slightly easier to add/remove merger rules in future.